Gems/MiniAudio/Code/Source/Clients/MiniAudioPlaybackComponentController.cpp

```diff
@@ -232,6 +340,22 @@ namespace MiniAudio
                     ma_sound_set_min_distance(m_sound.get(), m_config.m_minimumDistance);
                     ma_sound_set_max_distance(m_sound.get(), m_config.m_maximumDistance);
                     ma_sound_set_attenuation_model(m_sound.get(), static_cast<ma_attenuation_model>(m_config.m_attenuationModel));
+                    ma_sound_set_directional_attenuation_factor(m_sound.get(), m_config.m_directionalAttenuationFactor);
+
+                    // Set the forward direction for the sound source
+                    AZ::Transform worldTm = AZ::Transform::CreateIdentity();
+                    AZ::TransformBus::EventResult(worldTm, m_entityComponentIdPair.GetEntityId(), &AZ::TransformBus::Events::GetWorldTM);
+                    const AZ::Vector3 forward = worldTm.TransformVector(AZ::Vector3::CreateAxisY(1.f));
+                    if (!m_config.m_fixedDirection)
+                    {
+                        ma_sound_set_direction(m_sound.get(), forward.GetX(), forward.GetY(), forward.GetZ());
+                    }
+                    else
+                    {
+                        ma_sound_set_direction(m_sound.get(), m_config.m_direction.GetX(), m_config.m_direction.GetY(), m_config.m_direction.GetZ());
+                    }
```

Comment:
you should be able to use the same logic as above to only query and set the direction if not using m_fixedDirection